ui/src/components/virtual-scroll/QVirtualScroll.json

```diff
@@ -27,6 +27,16 @@
       "category": "content"
     },
 
+    "column-count": {
+      "type": "Number",
```

Comment:
Change it to `[ "Number", "String" ],`
rename it to `virtualScrollColumnCount` and move it to mixins/virtual-scroll.json